applications/asset_tracker_v2/overlay-debug.conf

```diff
@@ -16,6 +16,9 @@
 # 4 DEBUG, maximal level set to LOG_LEVEL_DBG
 CONFIG_LOG_MAX_LEVEL=4
 
+# Increase log buffer size to prevent logs being dropped.
+CONFIG_LOG_BUFFER_SIZE=2304
```

Comment:
Any reason for this number? May be it is better to set to 2048?

Reply:
Yes, 2048 wasn't enough :)